src/allmydata/scripts/tahoe_ls.py

```diff
@@ -64,13 +64,14 @@ def list(options):
         print(quote_output(data, quotemarks=False), file=stderr)
         return 1
 
+    path = unicode(path, "utf-8")
     nodetype, d = parsed
     children = {}
     if nodetype == "dirnode":
         children = d['children']
     else:
         # paths returned from get_alias are always valid UTF-8
```

Comment:
This comment should be moved up to line 67ish

Reply:
Turns out `escape_path()` can take both unicode and bytes, and that branch is actually covered. But yeah should be moved.